variables.tf

```diff
@@ -597,3 +597,23 @@ variable "kdump_crash_kernel_memory" {
   description = "The crashkernel memory reservation for kdump occurs during the system boot"
   default     = "2G-4G:384M,4G-16G:512M,16G-64G:1G,64G-128G:2G,128G-:4G"
 }
+
+################################################################
+# Secure NFS variables
+################################################################
+
+variable "secure_named" {
+  type        = bool
+  description = "Set to true to enable the secure_named"
+  default     = false
+}
+variable "secure_http" {
+  type        = bool
+  description = "Set to true to enable the secure_http"
+  default     = false
+}
+variable "secure_nfs" {
+  type        = bool
+  description = "Set to true to enable the secure_nfs"
```

Comment:
```suggestion
  description = "Flag to enable the secure_nfs which limits the access of NFS server to the compute/control plane"

Reply:
@prb112 I see this security guide says not to use `no_root_squash` and that is what this flag does. https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/4/html/security_guide/s2-server-nfs-noroot
Can you explain it to me, please? JFMI

Reply:
Hey Yussuf

Thanks for the review.

We based this off the /etc/exports configuration in the FYRE OCP Power environment.

`no_root_squash` means files created as PVC maintain their original uid/group ids instead of being 'protected'. This can have negative impacts to services like databases which may have privileged files. 

An example would be the Cloud Pak for Data https://www.ibm.com/docs/en/cloud-paks/cp-data/5.0.x?topic=storage-configuring-your-nfs-server 
> A Network Attached Storage (NAS) or Network File System (NFS) share must be exported to all NFS clients by using the following export options: rw, sync, no_root_squash, no_all_squash. 

So far, I haven't had to use `no_all_squash`.

Thanks,

Paul